testing/dart/canvas_test.dart

```diff
@@ -1031,6 +1031,38 @@ void main() async {
     canvas.restoreToCount(canvas.getSaveCount() + 1);
     expect(canvas.getSaveCount(), equals(6));
   });
+
+  test('TextDecoration renders non-solid lines', () async {
+    final File file = File(path.join('flutter', 'testing', 'resources', 'RobotoSlab-VariableFont_wght.ttf'));
```

Comment:
So I bet this path is wrong. There is some logic in here to look up the out directory from a system environment variable. We use that in some of the shader tests, let me find it

Reply:
See https://github.com/flutter/engine/blob/main/testing/dart/shader_test_file_utils.dart

You can get the build directory from the system env variable, and then the fixtures should be available under that.

Reply:
Well, that makes the local build even more annoying (another thing I have to magically set or it fails). But thanks!